test/blocklib/core/fft/fft.hpp

```diff
@@ -1,153 +1,63 @@
 #ifndef GRAPH_PROTOTYPE_FFT_HPP
 #define GRAPH_PROTOTYPE_FFT_HPP
 
+#include "../../algorithm/fft/fft.hpp"
```

Comment:
Hardcoding the relative dependency isn't great. The directories should be included into the default cmake (& meson)build includes ... makes it easier to refactor/shift later.